no-computer.md

```diff
@@ -11,41 +11,74 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I believe Lego can be a DSL. The need for Lego is met by people's desires to
```

Comment:
I totally agree&mdash;when talking about this course to friends and family explaining what a DSL can be, I've been referencing legos as a non-traditional DSL.